metrics/mir_team_bugs.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+"""Submit metrics for MIR Team queue sizes.
+
+Copyright 2019 Canonical Ltd.
+Mathieu Trudel-Lapierre <mathieu.trudel-lapierre@canonical.com>
+
+Based on triage.py:
+Joshua Powers <josh.powers@canonical.com>
+"""
+import argparse
+
+from metrics.helpers import lp
+from metrics.helpers import util
+
+
+def collect(dryrun=False):
+    """Submit data to Push Gateway."""
+    unassigned = lp.get_team_unassigned_bugs(team='ubuntu-mir',
+                                             distro='Ubuntu')
```

Comment:
I don't think it matters but in your lp changes use lower case Ubuntu.

Reply:
Fixed. Looks like most others used capitalized Ubuntu for other metrics.